src/TraceParserGen/TraceParserGen.cs

```diff
@@ -945,20 +926,16 @@ private string GetType(List<FieldInfo> versions)
     /// </summary>
     public static string ToCSharpName(string input)
     {
-        if (input == null)
-        {
-            return null;
-        }
+        if (String.IsNullOrEmpty(input))
+            return input;
 
-        for (int i = 0; i < input.Length; i++)
-        {
-            char c = input[i];
-            if (!Char.IsLetter(c) && !Char.IsDigit(c) && c != '_')
-            {
-                return Regex.Replace(input, @"[^\w\d_]", "");          //Simply remove them.    
-            }
-        }
-        return input;
+        string output = Regex.Replace(input, @"[^\w\d_]", "");
+
+        if (Char.IsDigit(output[0]))
```

Comment:
💡 

```suggestion
        if (char.IsDigit(output[0]))
```

Reply:
The rest of the code uses Char.IsDigit. Is there a reason to prefer char here?

Reply:
Sorry, i see now it is mixed. If you guys prefer this convention I'll stick to it.